packages/controller/lib/setup.js

```diff
@@ -521,7 +522,7 @@ async function processCommand(command, args, params, callback) {
                         createInitialInstances()
                             .then(() => new Promise(resolve => {
                                 // Creates a fresh certificate
-                                const Cert = require('./cli/cliCert');
+                                const Cert = require('@iobroker/js-controller-cli').command.cert;
```

Comment:
```suggestion
                                const Cert = cli.command.cert;
```
`cli` wird oben eh importiert